plugins/ltac/leminv.ml

```diff
@@ -257,9 +257,15 @@ let add_inversion_lemma_exn ~poly na com comsort bool tac =
 let lemInv id c =
   Proofview.Goal.enter begin fun gls ->
   let env = Proofview.Goal.env gls in
+  let clause = mk_clenv_from env (project gls) (c, pf_get_type_of gls c) in
+  let mv = let mvs = clenv_arguments clause in
+    if List.is_empty mvs then
+      CErrors.user_err
+        Pp.(hov 0 (pr_econstr_env (pf_env gls) (project gls) c ++ spc ()
```

Comment:
What's the point of creating this lovely error message if  the first thing we do with it is to discard it in the handler?